test/e2e/util/fixture.go

```diff
@@ -183,6 +183,7 @@ func renderCreate(ctx context.Context, opts *core.RawCreateOptions, platformOpts
 
 	opts.Render = true
 	opts.RenderInto = outputFile
+	opts.RenderSensitive = false
```

Comment:
Will this get overridden in core.CreateCluster below ?

Reply:
It was by the complete function but it should anymore.